src/introspection/getSchemaFromData.js

```diff
@@ -140,10 +141,29 @@ export default (data) => {
                 },
                 {}
             );
+
+            // Build input type.
+            var inputFields = { ...typeFields };
```

Comment:
this is a shallow clone. By calling `delete` below, I suspect you're updating the source object